en/02_Developer_Guides/06_Testing/04_Fixtures.md

```diff
@@ -139,19 +139,12 @@ seen by the fields prefixed with `=>`.
 Each one of our Players has a relationship to a Team, this is shown with the `Team` field for each `Player` being set
 to `=>App\Test\Team.` followed by a team name.
 
-[info]
-Take the player John in our example YAML, his team is the Hurricanes which is represented by `=>App\Test\Team.hurricanes`. This
+Take the player John in our example YAML below, his team is the Hurricanes which is represented by `=>App\Test\Team.hurricanes`. This
 sets the `has_one` relationship for John with with the `Team` object `hurricanes`.
-[/info]
```

Comment:
Took the content out of the callout because it's not really warranted, and it was violating [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.